airbyte-webapp/src/components/EntityTable/ConnectionTable.tsx

```diff
@@ -81,14 +82,10 @@ const ConnectionTable: React.FC<IProps> = ({ data, entity, onClickRow, onChangeS
     () => [
       {
         Header: (
-          <>
+          <th className={styles.tableHeader} onClick={() => onSortClick("name")}>
             <FormattedMessage id="tables.name" />
-            <SortButton
-              wasActive={sortBy === "name"}
-              lowToLarge={sortOrder === SortOrderEnum.ASC}
-              onClick={() => onSortClick("name")}
-            />
-          </>
+            <SortIcon wasActive={sortBy === "name"} lowToLarge={sortOrder === SortOrderEnum.ASC} />
```

Comment:
Previously this one was a button with icon(checked with VoiceOver):
<img width="685" alt="Screenshot at Sep 21 18-48-24" src="https://user-images.githubusercontent.com/20929344/191551683-0b037bfd-0641-4be7-894a-ecedcc73b1e6.png">